yesod-test/Yesod/Test.hs

```diff
@@ -162,6 +164,8 @@ import GHC.Exts (Constraint)
 type HasCallStack = (() :: Constraint)
 #endif
 
+{-# DEPRECATED byLabel "This function seems to have multiple bugs. Use byLabelExact instead" #-}
```

Comment:
How about including a link to this PR in the deprecation notices?

Reply:
It sounds great.